style.css

```diff
@@ -335,43 +352,205 @@ h1 {
 }
 
 .collapse ul li {
-  background-color: #f7f7f9;
-  font-family: 'Poppins', sans-serif;
-  font-style: normal;
+  background-color: var(--collapsebg);
   font-weight: 500;
   font-size: 15px;
   line-height: 20px;
-  padding: 10px;
-  margin: 5px 0 5px 0;
+  padding: 12px;
+  margin: 6px 0 6px 0;
 }
 
 .collapse ul li img {
   width: 48px;
   height: 48px;
   margin-right: 3px;
   vertical-align: middle;
-
-  /* text-top, text-bottom, sub, super, middle */
 }
 
 .aside-2 {
-  border-bottom-left-radius: 15px;
-  border-bottom-right-radius: 15px;
-  padding: 10px;
   order: 3;
   width: 100%;
 }
 
-.subtitle {
-  width: 295px;
-  height: 36px;
-  font-family: 'Poppins', sans-serif;
-  font-style: normal;
-  font-weight: 700;
-  font-size: 32px;
-  line-height: 44px;
-  color: #172b4d;
+/* contact */
+input {
+  width: 90%;
+  display: inline;
+  height: 48px;
+  background: var(--white);
+  border: 1px solid var(--info);
+  border-radius: 8px;
+  margin: 10px 0 10px 0;
+  padding: 15px;
+  font-size: 17px;
+  line-height: 20px;
+  align-items: center;
+  color: var(--txtFieldColor);
+}
+
+textarea {
+  height: 170px;
+  padding: 15px;
+  border: 1px solid var(--info);
+  border-radius: 8px;
+  margin: 10px;
+  width: 90%;
+  font-size: 17px;
+  line-height: 20px;
+  align-items: center;
+  color: var(--txtFieldColor);
+}
+
+form button {
+  margin-left: 20px;
+}
+
+.btn {
+  font-weight: 500;
+  font-size: 17px;
+  display: flex;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: var(--purple);
   flex: none;
   order: 0;
   flex-grow: 0;
+  box-sizing: border-box;
+  flex-direction: row;
+  padding: 12px;
+  gap: 10px;
+  height: 48px;
+  background: var(--white);
+  border: 1px solid var(--purple2);
+  border-radius: 8px;
+  margin-top: 15px;
+  margin-bottom: 10px;
+}
+
+.btn:hover {
+  color: var(--white);
+  background: var(--purple);
+}
+
+.btn:active {
+  color: var(--white);
+  background: var(--activeButton);
+}
+
+/* desktop */
+@media only screen and (min-width: 768px) {
+  .side-menu,
+  .hamb {
+    display: none;
+  }
+
+  .header {
+    background-color: var(--white);
+    padding: 0 155px 0 155px;
+    display: grid;
+    grid-template-columns: 20% 80%;
+  }
+
+  .logo {
+    height: 80px;
+    width: 100%;
+  }
+
+  .nav {
+    background-color: var(--white);
+    position: sticky;
+    height: 80px;
+    max-height: 80px;
+    width: 100%;
+    top: 0;
+    text-align: end;
+  }
+
+  .menu li {
+    display: inline-flex;
+  }
+
+  .menu a {
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+    display: flex;
+    align-items: center;
+    text-align: center;
+    color: #344563;
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+  }
+
+  .item:nth-of-type(1) {
+    background-color: var(--white);
+    background-image: url(./images/imgbg.jpg);
+    padding: 260px 280px 0 280px;
+    border-bottom-left-radius: 100px;
+  }
```

Comment:
- [ ] Please, let's try to stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the info of the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the info of the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). Therefore I suggest you ensure that the `background image` is positioned properly to match the Figma design. This will help us to make the portfolio professional and user-friendly ☺️.

Your design 😎 
![image](https://user-images.githubusercontent.com/99394374/229752852-2aa53c0a-d9bd-443d-a788-8611d9ee12f2.png)

Figma design 😎 
![image](https://user-images.githubusercontent.com/99394374/229753010-8a0bb76d-ec22-4d39-a00a-4ee49ac6de6c.png)